app/controllers/cloud_object_store_container_controller.rb

```diff
@@ -19,10 +19,9 @@ def button
 
     process_cloud_object_storage_buttons(params[:pressed])
 
-    if !@flash_array.nil? && params[:pressed].ends_with?("delete")
-      javascript_redirect :action      => 'show_list',
-                          :flash_msg   => @flash_array[0][:message],
-                          :flash_error => @flash_array[0][:level] == :error
+    if !@flash_array.nil? && params[:pressed].ends_with?("delete") && @display != "cloud_object_store_objects"
+      session[:flash_msgs] = @flash_array.dup
+      javascript_redirect previous_breadcrumb_url
```

Comment:
@miha-plesko While this works perfectly when the Cloud Object Store Containers are accessed from the Storage Manager Summary screen because we have a valid `previous_breadcrumb_url`, it does not work when I try to delete an item directly from `cloud_object_store_container/show_list`.

```
----] F, [2017-04-24T14:38:50.130901 #40537:3fce76c292a8] FATAL -- : Error caught: [NoMethodError] undefined method `[]' for nil:NilClass
/Users/aparnakarve/rh/master/manageiq/plugins/manageiq-ui-classic/app/controllers/application_controller.rb:2061:in `previous_breadcrumb_url'
```

Note that there are no breadcrumbs on this (`cloud_object_store_container/show_list`) screen.